src/views/Timesheets/components/ShiftItem/index.jsx

```diff
@@ -26,7 +26,7 @@ import styles from './ShiftItem.module.css';
 
 const CustomTooltip = withStyles((theme) => ({
   tooltip: {
-    backgroundColor: theme.palette.common.black,
+    backgroundColor: theme.colorSchemes.light.palette.common.black,
```

Comment:
This is hard-coding the light theme, we shouldn't merge it.

Reply:
Yeah, I think this is legacy from Sharon's work, reverting for now :+1: